tests/functional/conftest.py

```diff
@@ -0,0 +1,43 @@
+import subprocess
+
+import pytest
+
+
+@pytest.fixture(scope="session", autouse=True)
+def install_charmed_openstack_exporter_snap():
+    """Install the snap and enable openstack-exporter service for testing."""
+    snap_build_name = "charmed-openstack-exporter_*.snap"
+    config_dir_path = "/etc/openstack"
+    config_file_path = f"{config_dir_path}/clouds.yaml"
+    config_test_file_path = f"{config_dir_path}/test.yaml"
+
+    subprocess.check_call(
```

Comment:
**suggestion (non-blocking):** 

What if the snap file not exists, can we raise an error here?